loader/loader.c

```diff
@@ -536,19 +536,10 @@ static VkResult loader_add_instance_extensions(const struct loader_instance *ins
     }
 
     for (i = 0; i < count; i++) {
-        char spec_version[64];
-
         bool ext_unsupported = wsi_unsupported_instance_extension(&ext_props[i]);
         if (!ext_unsupported) {
-            (void)snprintf(spec_version, sizeof(spec_version), "%d.%d.%d", VK_VERSION_MAJOR(ext_props[i].specVersion),
-                           VK_VERSION_MINOR(ext_props[i].specVersion), VK_VERSION_PATCH(ext_props[i].specVersion));
-            loader_log(inst, VULKAN_LOADER_DEBUG_BIT, 0, "Instance Extension: %s (%s) version %s", ext_props[i].extensionName,
-                       lib_name, spec_version);
-
             res = loader_add_to_ext_list(inst, ext_list, 1, &ext_props[i]);
             if (res != VK_SUCCESS) {
-                loader_log(inst, VULKAN_LOADER_ERROR_BIT, 0,
```

Comment:
Is there a reason you removed the error?  It might be helpful

Reply:
I think it was because neither of the other places `loader_add_to_ext_list` is called report an error, and because all of the error paths in `loader_add_to_ext_list` log a message if a failure happened. The side reason is that the only reason this function would fail is if allocation failed.